### ISSUE TITLE
Add support for `loadFont`

### ISSUE THREAD

**Issue by aardvarkk** [sjoerd108/jimp-native#4]
Sorry if I'm missing something, but it's my understanding that `loadFont` is not part of the `jimp-native` library. I've noticed that bitmap font loading is *very slow* in `jimp`, and it looks like even if you try to run a bunch of font loading concurrently, they all get stuck in a queue anyway and the situation doesn't improve.

I think we'd see huge gains by implementing the font loading natively (and in parallel!) and this would eliminate a huge bottleneck in terms of speed.

**Comment by aardvarkk** [comment]
Further profiling indicates `loadFont` isn't as slow as I initially thought, so please feel free to close if this issue is irrelevant or not in the plan.

**Comment by sjoerdthedev** [comment]
I usually keep fonts around in memory and reuse them, if you have a lot of fonts that may not work though. The main bottleneck would come from reading and parsing the bitmaps themselves. I'm planning to overhaul this library so that parsing of image formats themselves are handled in C++ land, that would probably resolve the issue you're having.
